setup.py

```diff
@@ -2,19 +2,14 @@
 
 setup(
     name='pytmpdir',
-    packages=find_packages(exclude=['contrib', 'docs', 'tests']),  # this
-    # must be the same as the name
-    # above
+    packages=find_packages(exclude=['contrib', 'docs', 'tests']),
     version='0.1.0',
     description='A class representing a file system directory, that deletes on '
                 'garbage collect.',
     author='Synerty',
     author_email='support@synerty.com',
-    url='https://github.com/Synerty/pydirectory',
-    # use the URL to the github repo
-    download_url='https://github.com/Synerty/pydirectory/tarball/0.1.0',
-    # I'll explain this in a second
+    url='https://github.com/Synerty/pytmpdir',
+    download_url='https://github.com/Synerty/pytmpdir/tarball/0.1.0',
```

Comment:
See this file for how the pipbuild script auto updates these
https://github.com/Synerty/peek_worker/blob/master/setup.py#L28

I just run `./pipbuild.sh <newver>`